CRADLE/CorrectBias/correctBias.py

```diff
@@ -162,7 +162,7 @@ def run(args):
 	)
 	vari.HIGHRC = rc90Percentile
 
-	trainingSetMeta = utils.process(min(11, commonVari.NUMPROCESS), utils.fillTrainingSetMeta, trainingSetMeta)
+	trainingSetMeta = utils.process(min(11, commonVari.NUMPROCESS), utils.fillTrainingSetMeta, trainingSetMeta, context="fork")
```

Comment:
Curious about what changed after adding `context="fork"`

Reply:
The standard context/start method is set to "fork" in `cradle`: https://github.com/ReddyLab/CRADLE/blob/main/bin/cradle#L129. This is because the code in correctBias and callPeaks depends on the behavior of fork -- fork copies everything in memory from the parent process.

But "fork" has some problems. See https://pythonspeed.com/articles/python-multiprocessing/ for an explanation of them. Because of these issues "spawn" is a better method for starting new processes. So I updated the `process` method to use "spawn" be default, but added a parameter in case the old "fork" behavior was needed. "spawn" probably was actually fine here, but "fork" works too.

Reply:
See also https://docs.python.org/3/library/multiprocessing.html for more information on spawn/fork